modules/microshift-adding-olm-to-blueprint.adoc

```diff
@@ -0,0 +1,30 @@
+// Module included in the following assemblies:
+//
+// * microshift/microshift-update-rpms-ostree.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="adding-olm-to-blueprint_{context}"]
```

Comment:
```suggestion
[id="microshift-adding-olm-to-blueprint_{context}"]
```
I start every id with "microshift" because I've had sooo many broken builds.